packages-next/fields/src/types/timestamp/index.ts

```diff
@@ -2,13 +2,13 @@ import { DateTimeUtc } from '@keystonejs/fields';
 
 import type { FieldConfig } from '../../interfaces';
 import type { FieldType } from '@keystone-next/types';
-import type { BaseGeneratedListTypes } from '@keystone-next/types';
+import type { BaseGeneratedListTypes, FieldDefaultValue } from '@keystone-next/types';
 import { resolveView } from '../../resolve-view';
 
 export type TimestampFieldConfig<TGeneratedListTypes extends BaseGeneratedListTypes> = FieldConfig<
   TGeneratedListTypes
 > & {
-  defaultValue?: string;
+  defaultValue?: FieldDefaultValue<string>;
```

Comment:
Not a comment about this but defaultValue should probably live in the FieldConfig since it works for every field.